www/src/pages/docs/node-apis.js

```diff
@@ -34,7 +34,12 @@ class NodeAPIDocs extends React.Component {
             calling remote APIs, etc.) you must either return a promise or use
             the callback passed to the 3rd argument. Gatsby needs to know when
             plugins are finished as some APIs, to work correctly, require
-            previous APIs to be complete first.
+            previous APIs to be complete first. See{` `}
```

Comment:
Tangential to this--@pieh what do you think if we remove the callback example in the node-apis page?